livy/src/main/java/org/apache/zeppelin/livy/BaseLivyInterprereter.java

```diff
@@ -229,7 +229,31 @@ private InterpreterResult getResultFromStatementInfo(StatementInfo stmtInfo,
     } else {
       //TODO(zjffdu) support other types of data (like json, image and etc)
       String result = stmtInfo.output.data.plain_text;
-      if (result != null) {
+      
+      // check table magic result first
+      if (stmtInfo.output.data.application_livy_table_json != null) {
+        StringBuilder outputBuilder = new StringBuilder();
+        boolean isLastColumn = false;
+        
+        for (Map header : stmtInfo.output.data.application_livy_table_json.headers) {
```

Comment:
sorry, my fault. It should be `notFirstColumn`. We don't add '\t' for the first column